### PR TITLE
Initial commit for finnhub integration. Move dangerous commands out

### PR DESCRIPTION
**Pull request by mynameisivan**

### CODE REVIEW

File: commands/stonks.js [hunk 1]
```diff
@@ -1,18 +1,40 @@
-exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
-  const msg = await message.channel.send("Ping?");
-  msg.edit(`Pong! Latency is ${msg.createdTimestamp - message.createdTimestamp}ms. API Latency is ${Math.round(client.ping)}ms`);
+
```
Comment (BrandonWLee) [inline commands/stonks.js:1]: Empty line?

File: commands/stonks.js [hunk 2]
```diff
@@ -1,18 +1,40 @@
...
+api_key.apiKey = config.finnhub_api_key; 
+const finnhubClient = new finnhub.DefaultApi();
+
+exports.run = async (client, message, args, level) => {
+  console.log(args);
+  const company = args[0];
+  console.log(company);
+
+  if(company) {
+    finnhubClient.quote("VEEV", (error, data, response) => {
```
Comment (BrandonWLee) [inline commands/stonks.js:15]: wow

Comment (BrandonWLee) [reply commands/stonks.js:15]: You should add a TODO/another ticket so we don't forget to not make this hardcoded

File: commands/stonks.js [hunk 3]
```diff
@@ -1,18 +1,40 @@
...
+  }
+
 };
 
 exports.conf = {
   enabled: true,
   guildOnly: false,
   aliases: [],
-  permLevel: "User"
+  permLevel: "Users"
```
Comment (BrandonWLee) [inline commands/stonks.js:32]: Reason for this change?

File: commands/stonks.js [hunk 4]
```diff
@@ -1,18 +1,40 @@
...
+
+exports.run = async (client, message, args, level) => {
+  console.log(args);
+  const company = args[0];
+  console.log(company);
+
+  if(company) {
+    finnhubClient.quote("VEEV", (error, data, response) => {
+      console.log(data);
+      message.channel.send(`\`\`\`\n
```
Comment (BrandonWLee) [inline commands/stonks.js:17]: I think it would be a good idea to print out the stock ticker as well.

File: commands/stonks.js [hunk 5]
```diff
@@ -7,20 +7,27 @@ api_key.apiKey = config.finnhub_api_key;
...
 Today's Open: ${data.o}
 Today's High: ${data.h}
 Today's Low: ${data.l}
 Current Price: ${data.c}
 Yesterday's Close: ${data.pc}\`\`\``);
+      }
     });
+      
+  } else {
+    message.reply(".stonks [ticker]")
```
Comment (BrandonWLee) [inline commands/stonks.js:30]: The `.` is configurable right?

File: commands/stonks.js (hunk 4 above)
Comment (mynameisivan) [reply commands/stonks.js]: Done

File: commands/stonks.js (hunk 3 above)
Comment (mynameisivan) [reply commands/stonks.js]: It should be `User`

File: commands/stonks.js (hunk 5 above)
Comment (mynameisivan) [reply commands/stonks.js:30]: True... I wont address this one tho. Too much work.

Comment (BrandonWLee) [reply commands/stonks.js:30]: Damn it. You're fired!

File: commands/stonks.js (hunk 2 above)
Comment (BrandonWLee) [reply commands/stonks.js]: I see you addressed this. Thank you.

**Pull request merged**